config/schemas/config.yaml

```diff
@@ -7409,6 +7409,53 @@ properties:
         examples:
           - default-timeout-minutes: "30"
         type: object
+  gpu:
+    title: GPU Config
+    description: Configure the GPU Operator and its dependencies
+    type: object
+    properties:
+      enabled:
+        title: GPU Operator Enabled
+        default: false
+        type: boolean
+      operator:
+        title: GPU Operator Config
+        description: Configure GPU Operator
+        type: object
+        properties:
+          resources:
+            $ref: '#/$defs/kubernetesResourceRequirements'
+          tolerations:
+            $ref: '#/$defs/kubernetesTolerations'
+          affinity:
+            $ref: '#/$defs/io.k8s.api.core.v1.Affinity'
+      nodeFeatureDiscovery:
+        title: Node Feature Discovery Config
+        description: Configure Node Feature Discovery
+        type: object
+        properties:
+          master:
+            title: Node Feature Discovery master Config
+            description: Configure Node Feature Discovery masters
+            type: object
+            properties:
+              resources:
+                $ref: '#/$defs/kubernetesResourceRequirements'
+              tolerations:
+                $ref: '#/$defs/kubernetesTolerations'
+              affinity:
+                $ref: '#/$defs/io.k8s.api.core.v1.Affinity'
+          worker:
+            title: Node Feature Discovery worker Config
+            description: Configure Node Feature Discovery workers
+            type: object
+            properties:
+              resources:
+                $ref: '#/$defs/kubernetesResourceRequirements'
+              tolerations:
+                $ref: '#/$defs/kubernetesTolerations'
+              affinity:
+                $ref: '#/$defs/io.k8s.api.core.v1.Affinity'
```

Comment:
Question: You missed the daemonset part here right?

Reply:
Yep, I forgot it. Pushed it now.